charts/backstage/templates/backstage-deployment.yaml

```diff
@@ -126,6 +127,9 @@ spec:
             {{- if .Values.backstage.extraEnvVars }}
             {{- include "common.tplvalues.render" ( dict "value" .Values.backstage.extraEnvVars "context" $) | nindent 12 }}
             {{- end }}
+          {{- if .Values.backstage.resources }}
+          resources: {{- include "common.tplvalues.render" (dict "value" .Values.backstage.resources "context" $) | nindent 12 }}
+          {{- end }}
```

Comment:
This is already defined above https://github.com/backstage/charts/pull/34/files#diff-3484be37a646b3db74d7bd80ecd11ae378e6a2626f939ceda79aba27c4eab8d2R102-R104, please remove.

Reply:
@tumido corrected